src/AspNetCoreModuleV2/CommonLib/debugutil.cpp

```diff
@@ -266,42 +266,99 @@ IsEnabled(
     return ( dwFlag & DEBUG_FLAGS_VAR );
 }
 
+void WriteFileEncoded(UINT codePage, HANDLE hFile, const LPCWSTR  szString)
+{
+    DWORD nBytesWritten = 0;
+    auto const encodedByteCount = WideCharToMultiByte(codePage, 0, szString, -1, nullptr, 0, nullptr, nullptr);
```

Comment:
Pass in `WC_ERR_INVALID_CHARS` for flags? See: https://docs.microsoft.com/en-us/windows/desktop/api/stringapiset/nf-stringapiset-widechartomultibyte

Reply:
We want to see debug logs even if they contain invalid chars.